packages/wonder-blocks-dropdown/src/components/__tests__/combobox.test.tsx

```diff
@@ -296,6 +305,109 @@ describe("Combobox", () => {
         expect(screen.getByRole("combobox")).not.toHaveFocus();
     });
 
+    it("should clear the value when the user presses the clear button (x)", async () => {
+        // Arrange
+        const userEvent = doRender(
+            <Combobox selectionType="single" value="option2">
+                <OptionItem label="option 1" value="option1" />
+                <OptionItem label="option 2" value="option2" />
+                <OptionItem label="option 3" value="option3" />
+            </Combobox>,
+        );
+
+        // Act
+        await userEvent.click(
+            screen.getByRole("button", {name: /clear selection/i}),
+        );
+
+        // Assert
+        expect(screen.getByRole("combobox")).toHaveValue("");
```

Comment:
**suggestion:** I know this is an established pattern. Just wanted to note that I would move the `screen.getByRole` into the `// Act` as a `const result =  screen.getByRole` so that the expectation is purely inert and just doing the expectation check. Not a big deal, just wanted to note it.